cmd/kube-gen/main.go

```diff
@@ -79,6 +80,8 @@ func parseFlags() {
 		"If not specified, all types will be returned")
 	flags.BoolVar(&showVersion, "version", false, "display version information")
 	flags.BoolVar(&watch, "watch", false, "watch for new events")
+	flags.StringVar(&nodeEnvvar, "node-envvar", "", "only watch pods in the node specified by this environment variable. "+
```

Comment:
Sorry, I know this isn't originally your PR, but I wonder if this should just be a flag that can also be set via an environment variable. Maybe something like:

```suggestion
	flags.StringVar(&nodeEnvvar, "node", os.Getenv("KUBEGEN_NODE"), "If specified, only watch pods on the specified node. "+
```

If I were implementing this today, I'd probably follow this pattern for all of the flags.

Reply:
Incorporated the feedback, thanks!